models/iris_with_standoffs/model.sdf

```diff
@@ -538,11 +538,11 @@
     <static>0</static>
 
     <!-- plugins -->
-    <plugin filename="libignition-gazebo-joint-state-publisher-system.so"
-      name="ignition::gazebo::systems::JointStatePublisher">
+    <plugin filename="libgz-sim-joint-state-publisher-system.so"
```

Comment:
We can drop the `lib` prefix and `.so` suffix on the library names, so

`libgz-sim-joint-state-publisher-system.so` to `gz-sim-joint-state-publisher-system`, etc.